b.sh

```diff
@@ -2,7 +2,7 @@ echo "Building client..."
 go build -o bin/client main/client.go
 
 echo "Building MAC service..."
-GOARCH="arm64" GOOS="darwin" go build -o bin/serviceapp main/service.go
+GOARCH="arm64" GOOS="darwin" go build -o bin/service.app main/service.go
```

Comment:
Please keep it as is, the binary with .app suffix crashes.

Reply:
Ok, np. BUt that's strange.